### PR TITLE
Connect BE and FE using ra-data-simple-rest

### PR DESCRIPTION
**Pull request by amirshadaab**

### CODE REVIEW

File: giftacart-backend/app/controllers/users_controller.rb [hunk 1]
```diff
@@ -39,13 +40,13 @@ def destroy
   end
 
   private
```
Comment (amirshadaab) [inline giftacart-backend/app/controllers/users_controller.rb:41]: Indentation changes only

File: giftacart-backend/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::API
+  after_action :apply_content_range_header
+
+  protected
+  def apply_content_range_header
+    response.headers['Content-Range'] = 'orders 0-24/319'
```
Comment (AnthonyBobsin) [inline giftacart-backend/app/controllers/application_controller.rb:6]: what's this for?

**Review by AnthonyBobsin — APPROVED** [review]
LGTM!

File: giftacart-backend/app/controllers/application_controller.rb (hunk 2 above)
Comment (amirshadaab) [reply giftacart-backend/app/controllers/application_controller.rb:6]: React expects us to provide a content range in the response header so it knows how to paginate. This is a standard way of exposing that.

**Pull request merged**